calico/security/comms/secure-metrics.md

```diff
@@ -12,7 +12,7 @@ to limit access to {{ site.prodname }}'s metrics endpoints.
 ## Prerequisites
 
 - {{site.prodname}} is installed with Prometheus metrics reporting enabled.
-- `calicoctl` is [installed in your PATH and configured to access the data store](../../getting-started/clis/calicoctl/install).
+- `calicoctl` is [installed in your PATH and configured to access the data store](../../maintenance/clis/calicoctl/install).
```

Comment:
I noticed in other places we got rid of the `../..` references and replaced them with `{{site.baseurl}}` is there a specific pattern we want to use in general?

Reply:
It is, but I'm treating that as out of scope for this PR